docs/Networking.asciidoc

```diff
@@ -154,8 +156,12 @@ IPADDR='10.0.2.2/15'
 STARTMODE='auto'
 ---------------
 
-wicked 0.6.23 and later has enhanced support for the creation and configuration of OpenvSwitch bridges,
-so the permanent configuration should look like this:
+wicked 0.6.23 and later has enhanced support for the creation and configuration of OpenvSwitch bridges. 
+
+[NOTE]
+In some cases (e.g. on Leap) can be needed to start the OpenvSwitch service before the Network service by modifying the OpenvSwitch service file /usr/lib/systemd/system/openvswitch.service, where create an entry `Before=network.service` and remove network.service from `After=network.service`. For reference see https://en.opensuse.org/Portal:Wicked/OpenvSwitch#Wicked_0.6.23.2B[this].
```

Comment:
One does not simply modify `/usr/lib/systemd/*`.
Either stop at `by modifying the OpenvSwitch service` and leave it to user or suggest using `systemctl edit openvswitch` or to manually copy service file to `etc/systemd/system/openwswitch.service`.